pkg/cmd/openshift-apiserver/openshiftapiserver/openshift_apiserver.go

```diff
@@ -509,6 +511,28 @@ func (c *completedConfig) withUserAPIServer(delegateAPIServer genericapiserver.D
 	return server.GenericAPIServer, &legacyStorageVersionMutator{version: userapiv1.SchemeGroupVersion, storage: storage}, nil
 }
 
+func (c *completedConfig) withOpenAPIAggregator(delegatedAPIServer *genericapiserver.GenericAPIServer) (*genericapiserver.GenericAPIServer, error) {
+	delegatedAPIServer.RemoveOpenAPIData()
```

Comment:
why is this called? This needs extensive docs. We looked into the why at KubeCon.

Reply:
commented above, but the apiserver passed to the openapi aggregation controller must not have openapi fields set